src/event_queue.hpp

```diff
@@ -38,6 +39,9 @@ struct postsynaptic_spike_event {
     }
 };
 
+using event_vector = std::vector<postsynaptic_spike_event>;
```

Comment:
Cosmetic, but: `event_vector` is a bit more specific than the name implies. Consider e.g. `pse_vector`? We've got `pse` as an alias for `postsynaptic_spike_event` elsewhere in the code.

Reply:
Done.

I was in two minds about the name when writing this: `pse` felt "too abbreviated", and `postsynaptic_spike_event` was too long winded.